pkg/cmd/openshift-controller-manager/controller/quota.go

```diff
@@ -47,12 +47,10 @@ func RunResourceQuotaManager(ctx ControllerContext) (bool, error) {
 	return true, nil
 }
 
-type ClusterQuotaReconciliationControllerConfig struct {
-	DefaultResyncPeriod            time.Duration
-	DefaultReplenishmentSyncPeriod time.Duration
-}
+func RunClusterQuotaReconciliationController(ctx ControllerContext) (bool, error) {
+	defaultResyncPeriod := 5 * time.Minute
```

Comment:
guess we never allow to tweak these? (just asking I don't see a reason we should do)

Reply:
> guess we never allow to tweak these? (just asking I don't see a reason we should do)

we never did before.